content_cafe.py

```diff
@@ -45,27 +49,31 @@ class ContentCafeCoverageProvider(BibliographicCoverageProvider):
     INPUT_IDENTIFIER_TYPES = [Identifier.ISBN]
     DATA_SOURCE_NAME = DataSource.CONTENT_CAFE
     
-    def __init__(self, collection, api=None, mirror=None, **kwargs):
+    def __init__(self, collection, api=None, replacement_policy=None, **kwargs):
         """Constructor.
 
         :param collection: A Collection.
         :param api: A ContentCafeAPI.
-        :param mirror: A MirrorUploader.
+        :param replacement_policy: A ReplacementPolicy.
```

Comment:
Every constructor that use to take a Mirror now takes a complete ReplacementPolicy.